curses/cursesterm.c

```diff
@@ -888,12 +898,29 @@ static int curses_terminal_waitevent(Tn5250Terminal* This) {
  *****/
 static int curses_terminal_getkey(Tn5250Terminal* This) {
     int key;
+    MEVENT event;
 
     key = getch();
 
     while (1) {
         switch (key) {
 
+        case KEY_MOUSE:
+            if (getmouse(&event) == OK) {
+                int nx, ny;
+                /* Clamp to display size or we trip an assert. */
+                ny = tn5250_display_height(This->data->display) <= event.y
+                         ? tn5250_display_height(This->data->display) - 1
+                         : event.y;
+                nx = tn5250_display_width(This->data->display) <= event.x
+                         ? tn5250_display_width(This->data->display) - 1
+                         : event.x;
```

Comment:
I think it would be better if we just ignored the event if it was outside the tn5250 view.